docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml

```diff
@@ -162,6 +168,28 @@
             </row>
 
             <row>
+              <entry><emphasis>FOLD<indexterm>
+                  <primary>FOLD</primary>
```

Comment:
Once you document the new rights/code-signing stuff you should probably note that use of ,FOLD on a service or function definition within it requires additional rights, and that typically for security reasons ,FOLD should only be enabled in signed modules.